lib/ttdrest/concerns/creative_types/video.rb

```diff
@@ -21,15 +21,18 @@ def create_video_creative(name, video_content, clickthrough_url, landing_page_ur
             "LandingPageUrl" => landing_page_url
           }
 
-          if !params[:right_media_offer_type_id].nil?
+          if params[:right_media_offer_type_id].present?
             video_data = video_data.merge({"RightMediaOfferTypeId" => params[:right_media_offer_type_id]})
           end
-          if !params[:ad_technology_ids].nil?
+          if params[:ad_technology_ids].present?
             video_data = video_data.merge({"AdTechnologyIds" => params[:ad_technology_ids]})
           end
-          if !params[:is_securable].nil?
+          if params[:is_securable].present?
             video_data = video_data.merge({"IsSecurable" => params[:is_securable]})
           end
+          if params[:third_party_tracking_tags].present?
+            video_data = video_data.merge({"ThirdPartyTrackingTags" => [ params[:third_party_tracking_tags] ] })
```

Comment:
I don't think this is where we want to merge this, according to their api docs we just want to add the ImageAttributes here